circuits/cpp/barretenberg/ts/package.json

```diff
@@ -21,18 +21,19 @@
   ],
   "scripts": {
     "clean": "rm -rf ./dest .tsbuildinfo.browser .tsbuildinfo.node",
-    "build": "yarn clean && yarn build:wasm && yarn build:ts:browser && yarn build:ts:node",
+    "build": "yarn clean && yarn build:wasm && yarn copy_wasm_dest && yarn build:ts:browser && yarn build:ts:node",
     "build:dev": "tsc -b --watch",
     "build:wasm": "cd ../cpp && cmake --preset wasm-threads && cmake --build --preset wasm-threads && cmake --preset wasm && cmake --build --preset wasm",
+    "copy_wasm_dest": "cp ../cpp/build-wasm/bin/barretenberg.wasm ./dest/ && cp ../cpp/build-wasm-threads/bin/barretenberg.wasm ./dest/barretenberg-threads.wasm",
```

Comment:
Doesn't the symlink solve this issue?

Reply:
aiming for same state one would expect from published package with actual file instead of links to avoid potential problems in different environments.